Floofbot/Modules/Fun.cs

```diff
@@ -52,10 +49,33 @@ public async Task AskEightBall([Summary("question")][Remainder] string question)
             };
             Random random = new Random();
             int randomNumber = random.Next(responses.Count);
+            return responses[randomNumber];
+        }
+
+        public static string RespondEightBall(string question)
+        {
+            Regex questionRe = new Regex(@"^.*\?$");
+            if(questionRe.IsMatch(question)) {
+                string response = Floofbot.Modules.Fun.EightBallResponse();
+                if (!string.IsNullOrEmpty(response))
+                {
+                    string reply = $"> {question}";
+                    reply += "\n";
+                    reply += $"{response}";
+                    return reply;
+                }
+            }
+            return "";
```

Comment:
you can return `string.Empty` here just to make it clear that you didn't forget to fill in anything between the quotes